ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-main.yaml

```diff
@@ -20,15 +20,10 @@ operator:
   - pullspec: REPLACE_IMAGE:latest
     with: pipeline:opendatahub-operator
 releases:
-  initial:
-    integration:
-      name: "4.14"
-      namespace: ocp
   latest:
-    integration:
-      include_built_images: true
-      name: "4.14"
-      namespace: ocp
+    release:
+      channel: stable
+      version: "4.16"
```

Comment:
Perhaps give 4.17 a try ? If the issue persist, then we can ask related engineer to take a deeper look.

Reply:
Looks to still be not working :/